### ISSUE TITLE
Upgrading to devenv 1.0 leads to JSON parse errors for most commands

### ISSUE THREAD

**Issue by aviaviavi** [cachix/devenv#1022]
**Describe the bug**

I tried upgrading to devenv 1.0 and it seems to have broken the tool for me locally. I'm on an m2 Macbook air. 

`devenv shell`, `devenv up`, etc all give me:

```
~/d/test ❯❯❯ RUST_BACKTRACE=full devenv shell
• Building shell ...
thread 'main' panicked at devenv/src/command.rs:292:26:
Failed to parse JSON: Error("missing field `trusted`", line: 1, column: 35)
```

**To reproduce**
Please provide an [Short, Self Contained, Correct (Compilable), Example](http://sscce.org) by [creating a gist](https://gist.github.com/) using `devenv.nix`, `devenv.yaml`, and optionally `devenv.lock`.

```
~/d/test ❯❯❯ devenv init
...
~/d/test ❯❯❯ RUST_BACKTRACE=full devenv shell
• Building shell ...
thread 'main' panicked at devenv/src/command.rs:292:26:
Failed to parse JSON: Error("missing field `trusted`", line: 1, column: 35)
stack backtrace:
   0:        0x1004cbdc4 - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::h61949099c17bb561
   1:        0x1004f2ba4 - core::fmt::write::he0c0819610fe7c82
   2:        0x1004b11c0 - std::io::Write::write_fmt::h14dadda6958822c3
   3:        0x1004cbc18 - std::sys_common::backtrace::print::h10166cbeffac9d38
   4:        0x1004cc93c - std::panicking::default_hook::{{closure}}::hfec7fca779e11f3b
   5:        0x1004cc6bc - std::panicking::default_hook::h26402d2c6670ffd0
   6:        0x1004ccfb0 - std::panicking::rust_panic_with_hook::hb00dd38969b5a277
   7:        0x1004ccd68 - std::panicking::begin_panic_handler::{{closure}}::hc86edf66ba485638
   8:        0x1004cbff8 - std::sys_common::backtrace::__rust_end_short_backtrace::h24f57ebe971b5eac
   9:        0x1004ccaf0 - _rust_begin_unwind
  10:        0x10050defc - core::panicking::panic_fmt::h21b3a72f47844886
  11:        0x10050dd68 - core::result::unwrap_failed::h8e0283cd97c597e0
  12:        0x1001718b4 - devenv::command::<impl devenv::App>::prepare_command::hd087eee8f5f7b945
  13:        0x10016e5b8 - devenv::command::<impl devenv::App>::run_nix::h14deafc56b9d0dc0
  14:        0x10017de24 - devenv::App::get_dev_environment::h1684114f6a899665
  15:        0x10017a238 - devenv::App::prepare_shell::he4bc06dc86dff043
  16:        0x1001746e0 - devenv::main::hac819be715dd157e
  17:        0x1001b9170 - std::sys_common::backtrace::__rust_begin_short_backtrace::hc889c72b4fafb017
  18:        0x1001ae770 - std::rt::lang_start::{{closure}}::h6dcfa256e8a69797
  19:        0x1004cc9dc - std::panicking::try::ha883230fd73a158c
  20:        0x1004afa2c - std::rt::lang_start_internal::ha86617696da0c619
  21:        0x10018a694 - _main
✔ Building shell in 3.5s.
```

**Version**
 
devenv 1.0.0 (aarch64-darwin)

**Comment by domenkozar** [comment]
Seems like you have very old Nix running, I'll work on a fix.

**Comment by domenkozar** [comment]
Can you try using `main` branch and report if it works?

**Comment by gotcha** [comment]
@domenkozar I have the same issue and would be happy to help testing the `main` branch. But have no idea how...

**Comment by domenkozar** [comment]
`nix profile install --accept-flake-config tarball+https://install.devenv.sh/main`

**Comment by gotcha** [comment]
@domenkozar Seems to work for me except for :

```
✨ devenv 1.0.0 is newer than devenv input in devenv.lock. Run `devenv update` to sync.
```

I guess the message above is due to using the ``main``  branch.

**Comment by domenkozar** [comment]
Great :)

**Comment by domenkozar** [comment]
Released 1.0.1

**Comment by aviaviavi** [comment]
Fix worked for me, thanks Domen!
